doc/source/serve/tutorials/deployment-graph-patterns/conditional.md

```diff
@@ -0,0 +1,72 @@
+# Pattern: Conditional
+
+This example shows how to control your graph's flow using conditionals. You can use this pattern to introduce a dynamic path for your requests to flow through.
+
+![pic](https://raw.githubusercontent.com/ray-project/images/master/docs/serve/deployment-graph/control_flow_based_on_user_inputs.svg)
+
+## Code
+
+```{literalinclude} ../../doc_code/conditional.py
+:language: python
+:start-after: __graph_start__
+:end-before: __graph_end__
+```
+
+:::{note}
+`combine` takes in intermediate values from the call graph as the individual arguments, `value1` and `value2`. You can also aggregate and pass these intermediate values as a list argument. However, this list will contain references the values, rather than the values themselves. You must explicitly use `ray.get` to get the actual values before using them.
```

Comment:
not that in a deployment using `ray.get` is an antipattern because it blocks the `asyncio` loop. we should be recommending to `await `instead. We should make this abundantly clear in the model composition user guide. Could you add a short subsection on it?

Reply:
I added a warning about this.